ci-operator/step-registry/lvms/upgrade/lvms-upgrade-commands.sh

```
@@ -0,0 +1,188 @@
+  - mirrors:
+    - quay.io/redhat-user-workloads/logical-volume-manag-tenant/lvm-operator-bundle
+    source: registry.redhat.io/lvms4/lvms-operator-bundle
+  - mirrors:
+    - quay.io/redhat-user-workloads/logical-volume-manag-tenant/lvms-must-gather
+    source: registry.redhat.io/lvms4/lvms-must-gather-rhel9
+  - mirrors:
+    - quay.io/redhat-user-workloads/logical-volume-manag-tenant/topolvm
+    source: registry.redhat.io/lvms4/topolvm-rhel9
+EOF
+
+    if [ $? -ne 0 ]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?.

(SC2181)